assets/js/app.js

```diff
@@ -10,47 +9,63 @@ import selectValue from './select_value.js'
 window.htmx = htmx
 
 // load htmx extensions
-require('htmx.org/dist/ext/ws.js');
+require('htmx.org/dist/ext/ws.js')
```

Comment:
try not to reset all padding. That makes a PR harder to read ;-)

Reply:
Using Prettier with default style rules. I'll keep it in mind.

Reply:
No it's good that you introduced this, we had formatting for Go code, but not yet for js

Reply:
I'm going to use the same rules from now on

Reply:
@netsensei Probably best to add Prettier as a devcontainer extension then

Reply:
I found I had user preferences that overruled certain defaults. Will recommit this file with default formatting rules applied.